datasets/tr/fcib/crawler.py

```diff
@@ -135,40 +132,66 @@ def crawl(context: Context):
     table = doc.find('.//table[@class="table table-bordered"]')
     category_urls = [link.get("href") for link in table.findall(".//a")]
 
-    found_links = set()
+    # Ensure exactly 4 links are found
+    if len(category_urls) != 4:
+        context.log.warning(
+            f"Expected to find 4 category URLs, but found {len(category_urls)}"
+        )
 
+    found_bcd_links = []
+
+    # Process each category URL
     for url in category_urls:
+        if "5madde_ing" in url:
+            # Skip the "A" section (UN Security Council list)
+            context.log.info("Skipping A section (UN Security Council list)")
+            continue
+
+        # Fetch the content of the section
         section_doc = fetch_html(context, url, unblock_validator, cache_days=3)
+        doc_links = section_doc.xpath('//a[contains(@href, ".xlsx")]')
 
-        # Determine whether to search for .docx or .xlsx based on URL
-        if "5madde_ing" in url:
-            doc_links = section_doc.xpath('//a[contains(@href, ".docx")]')
-        else:
-            doc_links = section_doc.xpath('//a[contains(@href, ".xlsx")]')
-
-        for doc_link in doc_links:
-            found_links.add(doc_link.get("href"))
-
-    expected_links = set([DOCX_LINK, *(link for link, _, _ in XLSX_LINKS)])
-    # Check if new links have appeared
-    new_links = found_links - expected_links
-    assert not new_links, f"Unexpected links found: {new_links}"
-    # Check if all expected links are found
-    missing_links = expected_links - found_links
-    assert not missing_links, f"Expected links not found: {missing_links}"
-
-    # the actual crawling part if all links are verified
-    context.log.info("Fetching data from the provided XLSX links")
-    for url, program, short in XLSX_LINKS:
-        context.log.info(f"Processing URL: {url}")
+        # Expect exactly one .xlsx link in each section
+        if len(doc_links) != 1:
+            context.log.warning(
+                f"Expected to find 1 .xlsx link in section {url}, but found {len(doc_links)}"
+            )
+
+        # Extract the link
+        doc_link = doc_links[0].get("href")
+
+        # Extract the prefix letter (e.g., B, C, D) from the filename or URL structure
+        prefix = url.split("/")[-1].split("-")[0].upper()
```

Comment:
This part isn't relevant to the HTML URL, is it?

```suggestion
        slug = url.split("/")[-1].upper()
```

Then a better name would also be slug, since it's the page slug rather than the section number or prefix